web/pgadmin/tools/sqleditor/utils/query_tool_preferences.py

```diff
@@ -133,6 +133,16 @@ def register_query_tool_preferences(self):
         )
     )
 
+    self.open_file_in_new_tab = self.preference.register(
+        'Options', 'open_in_new_tab',
+        gettext("Open the file in new tab?"), 'boolean',
```

Comment:
Rename 'Open the file in new tab?' to 'Open the file in a new tab?'.

Reply:
Done